src/api/main.py

```diff
@@ -32,6 +32,7 @@
         o.strip()
         for o in Path(Path(__file__).parent / "origins.txt").read_text().splitlines()
     ]
+    origins = ["*"]
```

Comment:
Would it be more secure to add the actual origins endpoint here?

Reply:
Yes, it would but created a circle dependency problem

The web aca container depends on the api container to get its endpoint injected and the api container depends on the web container to get its domain for CORS

something to improve clearly

Reply:
Something that dapps framework must support I assume

that’s the reason why I was a bit reluctant to go the sep container way initially, it adds a lot of issues like this

you quickly realize you need more infra to support microservices

We might have to use dapps eventually but it felt overkill for this session, maybe next event 😊

Or maybe azd has a way to handle this circle dependency challenges, something to look into

Reply:
Can't we setup the ACA endpoints first, then get the endpoints, then deploy? If done in that order we should be able to get the urls after step one?

Reply:
This is the kind of thing I had in mind when I mentioned azd

this should work yeah, assuming you can update the aca env after initial provisioning

Reply:
I think this is not blocking though, it's just about the CORS filter being too open at the moment but doesn't change the rest of the story. We can revisit and narrow down the CORS filter when we get the chance.